### DISCUSSION TITLE
SHKeeper.io Module for WooCommerce: Enable Cryptocurrency Payments on Your Store

### DISCUSSION THREAD

**Discussion by tifon27** [woocommerce/woocommerce#53483] [General]
If you’re looking to add cryptocurrency payment options to your WooCommerce store, the SHKeeper module provides a simple and effective solution. It’s a free and open-source, non-custodial processor that allows you to accept various cryptocurrencies without intermediaries or additional fees.

SHKeeper is a self-hosted cryptocurrency payment processor developed by [VSYS Host](https://vsys.host). Designed with privacy and security in mind, it offers merchants a straightforward way to accept crypto payments while maintaining full control over their funds. SHKeeper is compatible with multiple eCommerce platforms and provides a flexible and customizable solution for online stores.

**Key Features**

- **Non-Custodial**: You retain complete control over your funds with no involvement from third parties.
- **Multi-Currency Support**: Accept payments in a range of cryptocurrencies. SHKeeper offers a direct way to receive BTC, ETH, LTC, DOGE, XMR, XRP, TRX, BNB, MATIC, AVAX, USDT (ERC20, TRC20, BEP-20, Polygon, Avalanche), USDC (ERC20, TRC20, BEP-20, Polygon, Avalanche).
- **No Transaction Fees**: SHKeeper does not charge any additional fees for processing transactions.
- **Easy Integration**: Simple to install and configure, with options to embed payment buttons or QR codes.
- **Customizable Settings**: Adjust exchange rates, fees, and more to suit your business needs.

You can download the SHKeeper module for WooCommerce from the official [marketplace](https://wordpress.org/plugins/shkeeper-payment-gateway/). 
Or from [GitHub](https://github.com/vsys-host/wp-shkeeper-plugin)

**Comment by brentmackinnon** [comment]
Going to close this one because it's an ad, not a discussion related to the woocommerce repo.

**Comment by Glec345** [comment]
The integration of Cryptopayments was smooth and swift; thus, international transactions were easier to handle. It has simplified the payment process with our global partners. Weare really happy with the decision to use this service.
